src/main/java/org/broadinstitute/hellbender/engine/filters/LibraryReadFilter.java

```diff
@@ -28,6 +30,10 @@ public LibraryReadFilter( final SAMFileHeader header ) {
     @Override
     public boolean test( final GATKRead read ) {
         final String library = ReadUtils.getLibrary(read, samHeader);
-        return library != null && library.equals(libraryToKeep);
+        if (library == null) {
+            return false;
+        }
+
+        return libraryToKeep.stream().anyMatch(library::equals);
```

Comment:
@magicDGS Shouldn't this use `libraryToKeep.contains(library)`  instead a streaming match?  That seems like it would be more efficient.

Reply:
True! My bad! Changed in next commit.